wqflask/wqflask/show_trait/export_trait_data.py

```diff
@@ -17,7 +17,7 @@ def export_sample_table(targs):
     for sample_group in ['primary_samples', 'other_samples']:
         for row in sample_data[sample_group]:
             sorted_row = dict_to_sorted_list(row)
-            print("sorted_row is:", pf(sorted_row))
+            #print("sorted_row is:", pf(sorted_row))
```

Comment:
**chore:** Remove this line

I'm trying to push for the removal of print(and debug statements). However, it's okay to have them around when you are working on the code itself--- they serve as a nice instrumentation tool and let you inspect things.